### ISSUE TITLE
Error trying to configure.

### ISSUE THREAD

**Issue by lgertel** [butlerx/fastify-bookshelfjs#2]
Hello everyone,
I'm with a problem configuring bookshelf with fastify.

When trying to start my app:
`AssertionError [ERR_ASSERTION]: ifError got unwanted exception: select `1` - Unknown column '1' in 'field list'`

It's weird the backticket but I really don't know why this is happening.

My package.json:
```
{
   "name": "nomikos",
   "version": "1.0.0",
   "description": "Cognitive system to help vote and decision in lawsuits",
   "main": "index.js",
   "scripts": {
      "test": "standard && tap test/*.test.js test/*/*.test.js test/*/*/*.test.js",
      "start": "fastify start app.js",
      "dev": "fastify start -l info -P app.js",
      "lint": "standard --fix"
   },
   "author": "Lucas Gertel",
   "license": "ISC",
   "dependencies": {
      "fastify": "^1.2.1",
      "fastify-autoload": "^0.3.0",
      "fastify-bookshelfjs": "^1.0.1",
      "fastify-cli": "^0.16.1",
      "knex": "^0.14.6",
      "mysql2": "^1.5.3"
   },
   "devDependencies": {
      "standard": "^11.0.1",
      "tap": "^11.1.3"
   }
}
```

and my appjs:
```
'use strict'

const path = require('path')
const AutoLoad = require('fastify-autoload')

module.exports = function (fastify, opts, next) {

  const bookshelf = require('fastify-bookshelfjs')

  fastify.register(
    bookshelf,
    {
      client: 'mysql2',
      connection: {
        host: '127.0.0.1',
        user: 'root',
        password: 'r00t',
        database: 'nomikos',
      },
    },
    console.error,
  )

  // This loads all plugins defined in plugins
  // those should be support plugins that are reused
  // through your application
  fastify.register(AutoLoad, {
    dir: path.join(__dirname, 'plugins')
  })

  // This loads all plugins defined in services
  // define your routes in one of these
  fastify.register(AutoLoad, {
    dir: path.join(__dirname, 'services')
  })

  // Make sure to call next when done
  next()
}
```

**Comment by butlerx** [comment]
This is an issue with how I used knex to try to test for a database connection. i think i have a solution using a raw query, that I will need to test

**Comment by lgertel** [comment]
Need help for pr?

**Comment by butlerx** [comment]
should be fixed in v1.0.2, tested with sqlite and mysql
